src/PhpBrew/Command/InstallCommand.php

```diff
@@ -343,6 +343,8 @@ public function execute($version)
         $variantBuilder = new VariantBuilder();
         $configureOptions = $variantBuilder->build($build);
 
+        $build->putEnvVars();
```

Comment:
Why does having the build object properly initialized require calling an extra method on it? Can it be done by default?

Reply:
I am not sure, I will try to do it.

Reply:
Dones